### ISSUE TITLE
Remote Control another PC's Discord Instance

### ISSUE THREAD

**Issue by spdermn02** [spdermn02/TouchPortal_Discord_Plugin#27]
As a User I would like to use TouchPortal to control another PC's Discord Instance.

Woudl have to build an independent executable to run on that other host, that needs to be running and waiting on a connection on a configured port.

Add configuration to Discord Plugin to allow for configuring IP address and Port of that 2nd PC.

Basically replicate all functions of Discord in that alternate build, So this would then mean the local discord plugin is really only a client between Touch Portal and the Remote Discord plugin.

Maybe needs to be Remote Discord Plugin as an alternate to this one.

**Comment by spdermn02** [comment]
Could become irrelevant once single Device to multi-PC support would come to Touch Portal. But that is still on the backlog of development from their side without a date in sight.

**Comment by geekgarage** [comment]
since, as you state, it's on their thing to do, you doing it would be a colossal waste of time. It would make more sense to make the devs of TouchPortal aware that the multi-pc feature is needed so all plugins gain this feature instead of every single plugin dev need to make this happen only for their plugin. If enough users are vocal about the need for multi-pc support they will for sure make it happen sooner rather than later and it will make a ton more sense that the main software adds this feature rather than plugin devs.

**Comment by spdermn02** [comment]
I agree that it would be a waste. I know they know the requests for this sort of single to multi feature is definitely wanted. Sometimes I do think it would be fun to have something like this functional just to make everything work for a specific plugin. But as I don't have a ton of development free time this is for sure a low low priority for when I do have time.

Thanks for the feedback though! If you haven't, feel free to add to the ideas-suggestions channel on the Touch Portal discord that way they see your desire for multi-pc connection.
